### PR TITLE
Add Ubuntu 18.04

### PR DESCRIPTION
**Pull request by manics**
Install OMERO 5.6 on Ubuntu 18.04.

Requires:
- [x] https://github.com/ome/ansible-role-postgresql/pull/19
- [x] https://github.com/manics/ansible-role-postgresql-client
- [x] https://github.com/manics/ansible-role-python3-virtualenv
- [x] https://github.com/manics/omero-cli-certificates

Tag: New feature, `3.3.0`

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -71,13 +71,22 @@ omero_server_python_addons: []
...
+
 
 ######################################################################
 # Expert users only!
 ######################################################################
 
 omero_server_ice_version: "3.6"
 
+omero_server_python_requirements_ice_package:
+  RedHat: "https://github.com/ome/zeroc-ice-py-centos7/releases/download/\
```
Comment (jburel) [inline defaults/main.yml:85]: How to you see handling c8 and ubuntu 20.04
Redhat8?

if this is an option like that, better to introduce the convention now

File: defaults/main.yml [hunk 2]
```diff
@@ -101,6 +111,9 @@ omero_server_python_requirements:
   - numpy
   - pillow
   - tables
```
Comment (jburel) [inline defaults/main.yml:113]: tables is now included in omero-server.

File: molecule/python3-ubuntu1804/molecule.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    v: true
+    diff: true
+    # tags: [x]
+  playbooks:
+    # TODO: Use shared test playbooks
+    converge: ../resources/playbook-py3.yml
+  inventory:
+    host_vars:
+      omero-server-python3:
+        postgresql_version: "10"
```
Comment (jburel) [inline molecule/python3-ubuntu1804/molecule.yml:37]: pg version 11 is the version we recommend so we should be testing it

File: defaults/main.yml (hunk 1 above)
Comment (manics) [reply defaults/main.yml:85]: The top level variable `omero_server_python_requirements_ice_package` will be the same, the keys will change but as you can see this is `Expert users only!` so we can do that later

File: defaults/main.yml (hunk 2 above)
Comment (manics) [reply defaults/main.yml:113]: It's not in https://github.com/ome/omero-cli-server/blob/v0.1.0/setup.py#L51-L55

File: molecule/python3-ubuntu1804/molecule.yml (hunk 3 above)
Comment (manics) [reply molecule/python3-ubuntu1804/molecule.yml]: Updated

File: defaults/main.yml (hunk 2 above)
Comment (jburel) [reply defaults/main.yml:113]: true only if you use ``[default]``

File: defaults/main.yml (hunk 1 above)
Comment (jburel) [reply defaults/main.yml:85]: as long as it does not require a major version bump for that adjustment

Comment (manics) [reply defaults/main.yml:85]: It won't be. The public "API" is in the README, this is considered internal.

**Comment by jburel** [comment]
Thanks for the changes
Looks good

**Comment by manics** [comment]
Merging for release

**Pull request merged**